pkg/config/testdata/piped/piped-config.yaml

```diff
@@ -121,7 +121,7 @@ spec:
           hookURL: https://slack.com/prod
       - name: integration-slack-api
         slack:
-          oauthToken: token
+          oauthTokenData: token
```

Comment:
```suggestion
          oauthToken: token
```
Please keep it.

Reply:
I think you should add new integrations(oauthTokenData and oauthTokenFile) for test.